doc/integration.rst

```diff
@@ -98,6 +100,10 @@ Parameters
 |                              |             |            | "fast": multi-cycle fast,                                       |
 |                              |             |            | "single-cycle": single-cycle                                    |
 +------------------------------+-------------+------------+-----------------------------------------------------------------+
+| ``ICache``                   | bit         | 0          | *EXPERIMENTAL* Add instruction cache instead of prefetch buffer |
++------------------------------+-------------+------------+-----------------------------------------------------------------+
+| ``ICacheECC``                | bit         | 0          | Add SECDED ECC protection to ICache (if ICache == 1)            |
```

Comment:
Should this also have an *EXPERIMENTAL* flag?

Reply:
Will add